store/postgres/migrations/2023-12-14-160628_change_health_column_format/down.sql

```diff
@@ -0,0 +1,23 @@
+
+create type subgraphs."health"
+    as enum ('failed', 'healthy', 'unhealthy');
+
+alter table
+    subgraphs.subgraph_deployment
+add
+    column health_new subgraphs.health;
+
+update
+    subgraphs.subgraph_deployment
+set
+    health_new = health::subgraphs.health;
+
+alter table
+    subgraphs.subgraph_deployment
+drop column
+    health;
+
+alter table
+    subgraphs.subgraph_deployment
+rename column
+    health_new to health;
```

Comment:
To put things back the way they were, I think this also needs a `not null` constraint.

Reply:
Yes, that's correct. It is fixed now.